pkg/cmd/pipelinerun/list_test.go

```diff
@@ -235,7 +247,7 @@ func TestListPipeline_empty(t *testing.T) {
 		t.Errorf("Unexpected error: %v", err)
 	}
 
-	test.AssertOutput(t, "No PipelineRuns found\n", output)
```

Comment:
this should have a `\n` when printing, was somthing removed?

Reply:
The formatting in the go templatewas changed, updated with correct test and formatting.